README.md

```diff
@@ -45,6 +45,63 @@ implemented and how they can be derived from new methods.
     - Two promises are equivalent when they yield equivalent values.
     - Two functions are equivalent if they yield equivalent outputs for equivalent inputs.
 
+## How to add Fantasy Land compatibility to your library
+
+1. Add `fanatasy-land` package as a peer dependency. Add in your `package.json`:
```

Comment:
I don't think so. For instance if we release two versions `1.0` with not prefixed names and `2.0` with `"fantasy-land/map"` like names. And a library will have a normal dependency  `fantasy-land@2.0.0`, we won't be able to use that library with an older library that uses unprefixed names. 

And with peer dependencies, we can kind of inject `fantasy-land@1.0.0` to that library.